src/redux-modules/settingsSlice.ts

```diff
@@ -145,6 +145,10 @@ export const settingsSlice = createSlice({
       set(state, `advanced.${statePath}`, value);
 
       handleAdvancedOptionsEdgeCases(state, statePath, value);
+
+      if (statePath == 'limitBatteryCharge') {
```

Comment:
does the battery limit setting persist on device reboot? or do we need to set it to 80% or 100% after every reboot?

Reply:
I only have rog ally with bazzite, and at least for this case, every reboot resets to 100%
That's something with 0 risk on write so it's required to apply after reboot